### ISSUE TITLE
Missing changelog entries > 3.4.0 in CHANGELOG.md

### ISSUE THREAD

**Issue by koobs** [tweepy/tweepy#1218]
Downstream OS packagers like to use upstream changelog links (using the relevant git tag for the changelog document) to provide users convenient breadcrumbs to changes in their commit log messages

CHANGELOG.md is missing entries since 3.4.0

I was unable to find an alternative changelog reference/link in the documentation. I considered that CHANGELOG.md may have been stale for this reason.

It would be great if the changelog could be brought up to date and updated for each release going forward before the new git version is tagged

Thanks!

**Comment by Harmon758** [comment]
Currently, the release notes for v3.4.0 and v3.5.0 can be found at https://github.com/tweepy/tweepy/releases/tag/v3.4.0 and https://github.com/tweepy/tweepy/releases/tag/v3.5.0, respectively.

Going back and adding changelogs/release notes for v3.6.0 and 3.7.0 is actually something I plan on doing, and I'll add the ones for v3.4.0 and v3.5.0 to CHANGELOG.md when I do so.

**Comment by koobs** [comment]
Much appreciated @Harmon758 and thanks for the reminder about releases

**Comment by Harmon758** [comment]
Resolved with https://github.com/tweepy/tweepy/commit/e794eeb0964caa7f0fa1c2d2c7607556165c6e9e, https://github.com/tweepy/tweepy/commit/664b74c9018786d81c078d438f37a1f22e1b78d3, and https://github.com/tweepy/tweepy/commit/85ed5fe87d770857f587cb588069aa6e945e4ce6.
I've also added the changelogs for v3.6.0 and v3.7.0 as release notes for the corresponding releases at https://github.com/tweepy/tweepy/releases.

As an off-topic housekeeping note, I've re-named the `3.7.0` tag to `v3.7.0` to be consistent with recent releases. As a result, the 3.7.0 release was re-published with the new release notes.
